xcom/README.md

```diff
@@ -0,0 +1,38 @@
+
+XCOM 2 is divided into two main environments: the "battlescape", where your squad of soldiers engages with a force of aliens at a specific location on the globe, and the "XCOM HQ", where you look after your soldiers, commission scientists to research new technologies to combat the aliens more effectively, and hire engineers to build the new technology.
+
+## Project Description
+
+In this project, we use Grakn to represent a queryable database of relevant data about XCOM technologies.
+
+In XCOM 2, technology must be researched to unlock new weapons and equipment. For example, when the [Beam Weapons](https://xcom.fandom.com/wiki/Beam_Weapons_(research)) research project is completed, the [Plasma Rifle](https://xcom.fandom.com/wiki/Plasma_Rifle_(XCOM_2)) becomes available for purchase. However, techs have prerequisites; you cannot begin research on Beam Weapons until you have finished research on [Magnetic Weapons](https://xcom.fandom.com/wiki/Magnetic_Weapons). Moreover, you cannot study the body of an Andromedon ([Andromedon Autopsy](https://xcom.fandom.com/wiki/Andromedon_Autopsy)) without first possessing an [Andromedon Wreck](https://xcom.fandom.com/wiki/Andromedon_Wreck).
+
+One use case for this data in-game is: we want to display a list of 'available research projects' - that is, the projects that are not completed, and for which we have met all of the prerequisites. We can ask Grakn: given the current state of the game (i.e: your current tech level and inventory), which techs are currently available for research?
+
+| ![research screen](images/xcom2-research-screen.jpg?raw=true) |
+|:--:|
+| ***There are currently 5 research projects available to work on.*** |
+
+Another potential use-case outside the game might be: a player wants to 'beeline' (i.e: rush) towards a certain critical tech, because they know they need it to build a powerful weapon. In this case, we can ask Grakn: what is the minimal set of techs required in order to unlock the Storm Gun?
+
+Storm Gun requires Beam Weapons, which requires Elerium and Magnetic Weapons; now, what do each of those require... You can certainly find out the answer by browsing the XCOM Wiki, but for a complex example, you may quickly find yourself getting lost in the tree of prerequisite techs. Enter Grakn, which, when the appropriate schema is loaded in, is able to fetch the correct answer in one simple query.
+
+| ![tech tree](images/tech-tree-mod.jpg?raw=true) |
+|:--:|
+| ***Instead of using Grakn, we could install a mod. But that's just not quite as cool, is it?*** |
+
+### Quickstart
+1. Install the latest version of [Grakn](https://github.com/graknlabs/grakn/releases) that is compatible with the latest version of Client Java (see the [dependency table](http://dev.grakn.ai/docs/client-api/java#dependencies))
```

Comment:
Should we dictate that users should always use the latest version. I think so, but is this consistent with the other examples in this repo?

Reply:
This quickstart step was, in fact, copied from the phone calls Java example.